torchvision/ops/_utils.py

```diff
@@ -67,3 +67,33 @@ def split_normalization_params(
         else:
             other_params.extend(p for p in module.parameters() if p.requires_grad)
     return norm_params, other_params
+
+
+def _upcast(t: Tensor) -> Tensor:
```

Comment:
Note that previous code used different versions of upcast. More specifically boxes permitted integers while losses didn't.